### PR TITLE
Update Creating_Solutions.md

### PR DESCRIPTION
**Pull request by jarusso**

### CODE REVIEW

File: Whats_New.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Keep up to date on what's new in IBM Watson Work Services. We'll list the latest
 
 | Date          | What's new       |
 | ------------- |:-------------|
+| 09.Aug.2018   | Graduated [message reactions](./guides/V1_wwsg_Reactions.md). from `EXPERIMENTAL` to `PUBLISHED` |
```
Comment (andrewegan) [inline Whats_New.md]: Assuming this should be space templates instead.

File: guides/Creating_Solutions.md [hunk 2]
```diff
@@ -441,3 +441,18 @@ When a user decides to create a new space of type Deal Room they will;
...
+- [Create a space template](./V1_create_space_template.md)
+- [Space template](./V1_space_template_main.md)
+- [Space template info](./V1_space_template_info.md)
+- [Get my space templates](./V1_get_my_space_templates.md)
+- [Get offering space templates](./V1_get_offering_space_templates.md)
+- [Get space template](./V1_get_space_template.md)
+- [Creat a space from a template](./V1_create_space_from_template.md)
+- [Delete space template](./V1_delete_space_template.md)
+- [Share space template](./V1_share_space_template.md) `EXPERIMENTAL`
+
```
Comment (andrewegan) [inline guides/Creating_Solutions.md:456]: Need to add the Fetch team templates link also.

File: NavigationMenu.md [hunk 3]
```diff
@@ -20,6 +20,16 @@
...
+  
+* ## [ Templates - Creating Solutions](./guides/Creating_Solutions.md) 
+  * [Create a space template](./guides/V1_create_space_template.md)
+  * [Space template](./guides/V1_space_template_main.md)
+  * [Space template info](./guides/V1_space_template_info.md)
+  * [Get my space templates](./guides/V1_get_my_space_templates.md)
+  * [Get offering space templates](./guides/V1_get_offering_space_templates.md)
+  * [Get space template](./guides/V1_get_space_template.md)
+  * [Creat a space from a template](./guides/V1_create_space_from_template.md)
+  * [Delete space template](./guides/V1_delete_space_template.md)
```
Comment (andrewegan) [inline NavigationMenu.md:33]: Need to add the Fetch team templates link also.

**Review by andrewegan — COMMENTED** [review]
Notes to self.

Need to delete from Coming Next page also.

**Comment by miguelestrada** [comment]
@andrewegan work with @jarusso to get this RTM and update when that is the case based on other work in backend.  Thx

File: Whats_New.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ Keep up to date on what's new in IBM Watson Work Services. We'll list the latest
 
 | Date          | What's new       |
 | ------------- |:-------------|
+| 09.Aug.2018   | Graduated [Templates](./guides/Creating_Solutions.md). from `EXPERIMENTAL` to `BETA` |
```
Comment (andrewegan) [inline Whats_New.md]: @a-kent Can you update here if I'm out of office.

**Pull request merged**